ui/src/translations/de.ts

```diff
@@ -151,39 +151,39 @@ export const de = {
     support_on_patreon: 'Auf Patreon unterstützen',
     general_sponsors:
       'Allgemeine Sponsoren sind die, die zwischen $10 und $39 zu Lemmy beitragen.',
-    crypto: 'Crypto',
+    crypto: 'Kryptographie',
```

Comment:
Sorry one more thing, this one refers to cryptocurrency (apparently "Kryptowährung" in German).

Reply:
Changed!